app/utils/ErrorHandler.scala

```diff
@@ -44,26 +43,23 @@ class ErrorHandler @Inject() (config: Configuration, auditConnector: AuditConnec
 
     implicit val headerCarrier: HeaderCarrier = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
 
-    logger.warn(
-      message = s"[ErrorHandler][onClientError] error in version " +
-        s"${Versions.getFromRequest(request).getOrElse("<unspecified>")}, " +
-        s"for (${request.method}) [${request.uri}] with status: " +
-        s"$statusCode and message: $message")
-    statusCode match {
+    logger.warn(message = s"[ErrorHandler][onClientError] error in version " +
```

Comment:
I think the previous version was a bit clearer, where "message = " is on the next line